ruby-training/trainees/alan/test_katas/01/math_maddness/math_maddness.rb

```diff
@@ -0,0 +1,26 @@
+require 'rspec/autorun'
+
+class Aritmetic_operations
+    
+    def self.add(first_number, second_number)
```

Comment:
The variable names are more descriptive than something like "num1" and "num2", but they're still to vague (what does the "number" represent?). In a calculator context, "numbers" are called/referred to as "operands" so "first_operand", "operand_one" or "left_operand" reads better. Please take this into advisement for all the parameters in this class and the future.